network/struct.go

```diff
@@ -107,6 +109,7 @@ func NewServerIdentity(public abstract.Point, address Address) *ServerIdentity {
 		Public:  public,
 		Address: address,
 		ID:      ServerIdentityID(uuid.NewV5(uuid.NamespaceURL, url)),
+		Cert:    []byte{},
```

Comment:
I changed `network.TLS(Cert|Key)PEM` to `type string`, so this problem does not show up anymore.
If you think a `NewTLSServerIdentity` makes sense, I'll add one. But to me a `NewServerIdentity` followed by a `si.Cert = cert` is OK.